addons/onboarding/models/onboarding_onboarding.py

```diff
@@ -74,6 +74,14 @@ def action_close(self):
         """Close the onboarding panel."""
         self.current_progress_id.action_close()
 
+    @api.model
+    def action_safe_close(self, xmlid):
```

Comment:
Have to say this is strange, as it adds yet another layer of methods while the code flow is already complicated. Are there places where we want the "unsafe" behavior ? Can't we make the safe behavior standard ?